src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,385 @@
+-   If a commercial cap such as EasyCap actiCAP 64 Ch Standard-2
+was used, then the values `CapManufacturer = "EasyCap"`, `CapManufacturersModelName = "actiCAP
+64 Ch Standard-2"` and `NIRSPlacementScheme = "n/a"` should be used.
+-   If an EasyCap was used but with custom positions,
+as may be done by cutting custom holes in the cap,
+then the values `CapManufacturer = "EasyCap"`, `CapManufacturersModelName = "custom"` and
+`NIRSPlacementScheme = "n/a"` should be used.
+-   If a completely custom cap was knitted, then
+`CapManufacturer = "custom"`, `CapManufacturersModelName = "custom"` and
+`NIRSPlacementScheme = "n/a"`.
+-   If no cap was used and optodes were taped to the scalp
+at positions Cz, C1 and C2, then the values `CapManufacturer = "none"`, `CapManufacturersModelName
+= "n/a"` and `NIRSPlacementScheme = "["Cz", "C1", "C2"]"` should be used.
+In these cases additional information regarding channels and optodes SHOULD be placed in `*_channels.tsv` and `*_optodes.tsv` files.
+
+Closely spaced or short-separation source-detector pairs are often included in fNIRS measurements to
+obtain a measure of systemic, rather than neural, activity. These source-detector
+pairs are referred to as *short channels*. There is variation in how manufacturers
+implement these short channels, some use specialised sources or detectors,
+and the placement mechanisms vary.
+It is beyond the scope of the BIDS specification to define what constitutes a short channel,
+and detailed characteristics of channels may be stored within the SNIRF file
+(for example, in the `sourcePower` field).
+However, to improve searchability and ease of access for users, it is useful to
+know if short channels were included in the fNIRS measurements; this information
+is stored in the field `ShortChannelCount`.
+If the field `ShortChannelCount` is populated, then the optional column `short_channel`
+may be used in `*_channels.tsv` to describe which channels were specified as short.
+
+Generic fields that MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "TaskName": "REQUIRED",
+   }
+) }}
+
+Generic fields which SHOULD be present: For consistency between studies and institutions,
+we encourage users to extract the values of these fields from the actual raw data.
+Whenever possible, please avoid using ad hoc wording.
+
+{{ MACROS___make_metadata_table(
+   {
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "Manufacturer": "RECOMMENDED",
+      "ManufacturersModelName": "RECOMMENDED",
+      "SoftwareVersions": "RECOMMENDED",
+      "TaskDescription": "RECOMMENDED",
+      "Instructions": "RECOMMENDED",
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+      "RecordingDuration": "RECOMMENDED",
+      "HeadCircumference": "RECOMMENDED",
+      "HardwareFilters": "RECOMMENDED",
+      "SubjectArtefactDescription": "RECOMMENDED",
```

Comment:
is this SubjectArtefactDescription field needed?

Reply:
the drop-down also still includes an EEG specific description: 
> Freeform description of the observed subject artefact and its possible cause (for example, "Vagus Nerve Stimulator", "non-removable implant"). If this field is set to "n/a", it will be interpreted as absence of major source of artifacts except cardiac and blinks.

Reply:
I think we did not draw a conclusion on this yet, as the specification was not altered so far. Should we reopen this for discussion?